src/CommandingOfficers/Qis.java

```diff
@@ -0,0 +1,234 @@
+    public instantiator()
+    {
+      super("Qis");
+      infoPages.add(new InfoPage(
+          "Commander Qis (pronounced 'keese') has spent his adult life fleeing cold weather, and failing to escape it.\n" +
+          "Whether the cause is simply incredible misfortune or some sort of curse is unclear... he is quite curt on the subject.\n" +
+          "What is known is that Qis has become incredibly adept at using the ever-present snow to his advantage."));
+      infoPages.add(new InfoPage(
+          "Passive:\r\n" +
+          "Watery terrain slows your non-naval units, but all units have perfect movement in snow.\n" +
+          "The path each unit takes on your turn is coated with snow until your next turn."));
+      infoPages.add(new InfoPage(
+          GroovinMovin.NAME+" ("+GroovinMovin.COST+"):\n" +
+          "+"+GroovinMovin.BUFF+" move for all units\n" +
+          "Snow trails left this turn last +1 turn for each space moved so far"));
+      infoPages.add(new InfoPage(
+          TrailOfChill.NAME+" ("+TrailOfChill.COST+"):\n" +
+          "+"+TrailOfChill.BUFF+" move for all units\n" +
+          "All snow on the map will last "+TrailOfChill.DURATION+" turns"));
+      infoPages.add(new InfoPage(
+          "Hit: Deserts, Desserts\n" +
+          "Miss: Showing up first"));
+      infoPages.add(new InfoPage(
+          "Original art credit:\n" +
+          "@Smile ❤#3047 Discord ID 374728627373211651"));
+    }
+    @Override
+    public Commander create(GameScenario.GameRules rules)
+    {
+      return new Qis(rules);
+    }
+  }
+
+  private int snowTrailTurns = 1;
+  private boolean snowTrailShouldScale = false;
+
+  public Qis(GameScenario.GameRules rules)
+  {
+    super(coInfo, rules);
+
+    addCommanderAbility(new GroovinMovin(this));
+    addCommanderAbility(new TrailOfChill(this));
+  }
+
+  @Override
+  public void modifyMoveType(UnitContext uc)
+  {
+    // Qis's units are fine in the snow, but not when wet.
+    if( !uc.model.isSeaUnit() )
+    {
+      // Rain gives +1 move cost across the board
+      for( TerrainType terrain : TerrainType.TerrainTypeList )
+        uc.moveType.setMoveCost(Weathers.RAIN, terrain, uc.moveType.getMoveCost(Weathers.CLEAR, terrain) + 1);
+
+      for( Weathers weather : Weathers.values() )
+      {
+        if( weather == Weathers.RAIN || weather == Weathers.SNOW )
+          continue; // These are handled elsewhere
+
+        // Watery terrain costs +1 move
+        for( TerrainType terrain : TerrainType.TerrainTypeList )
+        {
+          if( terrain.isWater() )
+            uc.moveType.setMoveCost(weather, terrain, uc.moveType.getMoveCost(weather, terrain) + 1);
+        }
+      }
+    }
+    // Perfect movement on snow
+    for( TerrainType terrain : TerrainType.TerrainTypeList )
+    {
+      if( MoveType.IMPASSABLE > uc.moveType.getMoveCost(Weathers.SNOW, terrain) )
+        uc.moveType.setMoveCost(Weathers.SNOW, terrain, 1);
```

Comment:
So his units can move better on snowy mountains than on clear mountains?

Reply:
Yes. I even accidentally let his tanks move on mountains accidentally.
The whole gameplay schtick here is that you move a unit through the heavy terrain to snow it for the dudes behind that dude. The effect is *sorta* like Sturm, but not if you're just YOLOing one unit (and it requires more planning)